ideas/310-core-team-proposal.md

```diff
@@ -72,72 +71,84 @@ See https://github.com/status-im/status-react/issues/6757 as an example of a sim
 
 The projects list should have links to umbrella issues for each projects.
 
-### Meetings & Philosophy
+## Research
 
-Process can be revisited through retrospective. Probably, the process should go to a github repository as a markdown file and using PRs to update it.
+### 0. Meta (what this team will be doing?)
 
-Taking as little of the parallel projects as possible. For example, we have to do maintenance anyway, so adding one other project as a focus area should be okay.
+Compeltion by December, 19th (1 week)
```

Comment:
typo

Reply:
Possibly a bit too tight for research phase? Depends on interaction with Onboarding actions, cc @rachelhamlin @hesterbruikman

Reply:
fixed the typo

Reply:
For the onboarding/polish actions, I agree its hard to plan in advance when bugs keep re-appearing. However there are probably 3-5 major areas that require significant planning in advance to fix rather then patches which reappear as something else in the future.

Reply:
yep, but I supposed that this (meta) proposal only identifies these areas, and then they need to be researched/spec'd/implemented separately